README.md

```diff
@@ -202,6 +202,15 @@ $acclimator->registerAdapter('Your\Favorite\ContainerAdapter', 'Your\Favorite\Co
 $adaptedContainer = $acclimator->acclimate($container);
 ```
 
+## Roadmap
+
+Plans for Acclimate 2.0.0 include the following:
+
+- Adding support of the latest versions of all relevant containers.  This includes Laravel, Zend, Pimple and Symfony.
```

Comment:
That's a fair point. @Sam-Burns Maybe you should do a 1.1 with the Guzzle stuff marked as deprecated (which means we'll have to add it back in). Seems like the other updates have been backward-compatible so far.